doc/source/tutorial/linalg_batch.md

```diff
@@ -0,0 +1,148 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
+---
+
+(linalg_batch)=
+# Batched Linear Operations
+
+Some of SciPy's linear algebra functions support N-dimensional array input. These operations have not been mathematically generalized to higher-order tensors; rather, the indicated operation is performed on a *batch* (or "stack") of input scalars, vectors, and/or matrices.
+
+Consider the `linalg.det` function, which maps a matrix to a scalar.
+
+```{code-cell} ipython3
+import numpy as np
+from scipy import linalg
+A = np.eye(3)
+linalg.det(A)
+```
+
+Sometimes we need the determinant of a batch of matrices of the same dimensionality.
+
+```{code-cell} ipython3
+batch = [i*np.eye(3) for i in range(1, 4)]
+batch
+```
+
+We could perform the operation for each element of the batch in a loop or list comprehension:
+
+```{code-cell} ipython3
+[linalg.det(A) for A in batch]
+```
+
+However, just as we might use NumPy broadcasting and vectorization rules to create the batch of matrices in the first place:
+
+```{code-cell} ipython3
+i = np.arange(1, 4).reshape(-1, 1, 1)
+batch = i * np.eye(3)
+batch
+```
+
+we might also wish to perform the determinant operation on all of the matrices in one function call.
+
+```{code-cell} ipython3
+linalg.det(batch)
+```
+
+In SciPy, we prefer the term "batch" instead of "stack" because the idea is generalized to N-dimensional batches. Suppose the input is a 2 x 4 batch of 3 x 3 matrices.
+
+```{code-cell} ipython3
+batch_shape = (2, 4)
+i = np.arange(np.prod(batch_shape)).reshape(*batch_shape, 1, 1)
+input = i * np.eye(3)
+```
+
+In this case, we say that the *batch shape* is `(2, 4)`, and the *core shape* of the input is `(3, 3)`. The net shape of the input is the sum (concatenation) of the batch shape and core shape.
+
+```{code-cell} ipython3
+input.shape
+```
+
+Since each 3 x 3 matrix is converted to a zero-dimensional scalar, we say that the core shape of the outuput is `()`. The shape of the output is the sum of the batch shape and core shape, so the result is a 2 x 4 array.
+
+```{code-cell} ipython3
+output = linalg.det(input)
+output
+```
+
+```{code-cell} ipython3
+output.shape
+```
+
+Not all linear algebra functions map to scalars. For instance, the `linalg.expm` function maps from a matrix to a matrix with the same shape.
```

Comment:
If you want to link to the `linalg.expm` docstring, you can use
```suggestion
Not all linear algebra functions map to scalars. For instance, the {func}`scipy.linalg.expm` function maps from a matrix to a matrix with the same shape.
```
You could also use similar syntax as in rst: 

```
{func}~.linalg.expm`
```
but in this particular case it is ambiguous since we have `scipy.linalg.expm` and also `scipy.sparse.linalg.expm` with the same role (func).